### ISSUE TITLE
SPAR user/role assigment import

### ISSUE THREAD

**Issue by gormless87** [bcgov/nr-forests-access-management#462]
**Describe the task**
Import a set of user assignments to roles from SPAR team

**Acceptance Criteria**
- [ ] users and roles imported
- [ ] confirm users can login
- [ ] confirm users have correct role assignments

**Additional context**
- SPAR team works with DBA to export users/roles from ADAM and resolve any data quality issues
- validated users provided by SPAR team
- related decision note to be written up:  How will users to be imported be validated --call the IDIM api for each entry OR send list to IDIM to validate?

**Comment by MCatherine1994** [comment]
All the SPAR roles are abstract role, which requires a forest client number when doing the user role assignment, but not all the SPAR users have a forest client number

**Comment by MCatherine1994** [comment]
Setup a meeting with Sabina to chat about the SPAR user and roles, wondering if the data is correct for those missing the forest client number, and also some records show "OBSOLETE" if we want to ignore those.

**Comment by MCatherine1994** [comment]
Got the script to read users from csv file and transfer into FAM. Need to understand more about the SPAR user role requirement, will let Olga lead to understand the SPAR requirement. Script should be ready.

**Comment by basilv** [comment]
I'm confused by the status of this. Is there anything to code review? I don't think its ready for code review, as we don't know what to do yet about the SPAR organizational units that aren't forest clients.

**Comment by ArogeG** [comment]
Blocked by the SPAR organizational unit that have forest client and those that don't .

**Comment by ArogeG** [comment]
Decision to repurpose this ticket for a generic script and then create a new ticket for the SPAR solution.
